Dockerfile

```diff
@@ -145,19 +161,19 @@ CMD gunicorn \
     --access-logfile \
     - ${SERVICE_VARIANT}.wsgi:application
 
+########### development target ###########
 FROM base as development
 
 COPY --from=builder-development /edx/app/edxapp/venvs/edxapp /edx/app/edxapp/venvs/edxapp
 
 USER root
-
 RUN ln -s "$(pwd)/lms/envs/devstack-experimental.yml" "$LMS_CFG"
 RUN ln -s "$(pwd)/cms/envs/devstack-experimental.yml" "$CMS_CFG"
 RUN touch ../edxapp_env
 
 USER app
 
-ENV EDX_PLATFORM_SETTINGS='devstack_docker'
+ENV EDX_PLATFORM_SETTINGS='devstack'
```

Comment:
I don't think we need to continue using devstack_docker here, based on https://github.com/openedx/edx-platform/blob/master/lms/envs/devstack_docker.py